docs/_includes/engage.html

```diff
@@ -1,124 +1,122 @@
-                <p class="card-text">
-                  Join the SMP Cloud Native Community Group!
-                </p>
-                </a>
-              </div>
-            </div>  
+        </div>
+        <!-- 2nd row -->
+        <div class="row">
+        <!-- Youtube -->
+        <div class="col-lg-4">
+          <div class="card-web card hover-up hover-line">
+            <div class="card-body">
+              <a href="https://www.youtube.com/c/Layer5io?sub_confirmation=1"><h4 class="card-title"><i class="fab fa-youtube yt"></i> YouTube</h4>
+              <p class="card-text">
+                Watch community meeting recordings and learn about distributed performance management. 
+              </p>
+              </a>
+            </div>
           </div>
-          <!-- Meeting Minutes -->
-          <div class="col-lg-4">
-            <div class="card-web card hover-up hover-line"> 
-              <div class="card-body">
-                <a href="https://bit.ly/3B7uIy9"><h4 class="card-title"><i class="far fa-file cmu"></i> Meeting Minutes</h4>
-                <p class="card-text">
-                 Have a topic to discuss? Add it to the meeting minutes.
-                </p>
-                </a>
-              </div>
-            </div>  
-          </div>          
         </div>
+        <!-- Community Calender -->
+        <div class="col-lg-4"> 
+          <div class="card-web card hover-up hover-line">  
+            <div class="card-body">    
+              <a href="https://calendar.google.com/calendar/b/1/r?cid=bGF5ZXI1LmlvX2VoMmFhOWRwZjFnNDBlbHZvYzc2MmpucGhzQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20"><h4 class="card-title"><i class="far fa-calendar-alt cal"></i> Community Calender</h4>
+              <p class="card-text">
+                  Subscribe to the community calender & stay updated. 
+              </p>
+              </a>
+            </div>
+          </div>         
+        </div>
+        <!-- Layer5 Community -->
+        <div class="col-lg-4">
+          <div class="card-web card hover-up hover-line"> 
+            <div class="card-body">
+              <a href="https://layer5.io/community"><h4 class="card-title"><i class="fas fa-users cmu"></i> Service Mesh Community</h4>
+              <p class="card-text">
+                Join the largest collection of service mesh projects and their maintainers in the world. 
+              </p>
+              </a>
+            </div>
+          </div>  
+        </div>
+        <!-- Community Meeting -->
+        <div class="col-lg-4">
+          <div class="card-web card hover-up hover-line"> 
+            <div class="card-body">
+              <a href="https://meet.layer5.io/smp"><h4 class="card-title"><i class="fas fa-headset cmu"></i> Biweekly Meeting</h4>
+              <p class="card-text">
+                Join the SMP Community Meetings biweekly on Thursdays at 9 - 10 AM Central.
```

Comment:
Is it should be updated to Layer5 Community instead of SMP Community?

Reply:
IDK. Haven't changed that thing.

Reply:
> Because I saw you changed so here:
> 
> https://github.com/service-mesh-performance/service-mesh-performance/pull/426/files#diff-c44914a3ea374b20e9087a2b61673850b872694b9c440ac83b229ba556350cb9L91-R39

Thanks. I'll do it.

Reply:
@gyohuangxin I checked it with live website, the text and order of elements is fine in the PR. The formatting and editing is make it confusing, kindly review it again.